### ISSUE TITLE
Revise def: GO:0048315 conidium formation

### ISSUE THREAD

**Issue by dianeoinglis** [geneontology/go-ontology#12743]
Current Def: "Formation of asexual, non-motile spores in fungi, by conversion of hyphal elements or arising from sporogenous cells"

Actually, conidia are produced asexually but conidia of compatible mating types can participate in sexual reproduction so, the current def is technically inaccurate although this exact phrase can be found in definitions on the web...just because it may be written somewhere on the web does not guaranty that it is true!

New proposed def: The process of producing non-motile spores through an asexual process of reproduction via mitosis, to produce new haploid cells that are genetically identical to the haploid parent. Conidia are produced by conversion of hyphal elements or are borne on sporogenous cells on or within specialized structures termed conidiophores and participate in dispersal of the fungus.

This proposed def is more wordy but also more informative. Cut any part that seems unnecessary as deemed fit. GO:0061794 'conidium development,' GO:0075306 'regulation of conidium formation,' GO: 0075306 'regulation of...,' GO:0054308 'negative reg...' GO:0054307 'positive reg...' and any others based on conidium formation or development should be similarly revised.

**Comment by paolaroncaglia** [comment]
@dianeoinglis 
Done. I opted for "The process of producing non-motile spores, called conidia, via mitotic asexual reproduction in higher fungi. Conidia are haploid cells genetically identical to their haploid parent. They are produced by conversion of hyphal elements, or are borne on sporogenous cells on or within specialized structures termed conidiophores, and participate in dispersal of the fungus.". I edited definitions of other conidium terms accordingly.
